ui-frontend/packages/catalog-ui-search/src/main/webapp/component/pages/incompatible-browser.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+const NoIE = require('../../img/no-ie.svg') as string
+const Edge = require('../../img/edge.svg') as string
+const Firefox = require('../../img/firefox.svg') as string
+const Chrome = require('../../img/chrome.svg') as string
+
+export const IncompatibleBrowser = () => {
```

Comment:
I wasn't sure where the best location to put these files would be. Is `component/pages` okay or is there a better place to put them?

Reply:
👍  I feel like this is as fine a place as any. The name of the file alone I think makes it easy to track down so I'm not too worried about it.